docroot/modules/humsci/hs_layouts/hs_layouts.module

```diff
@@ -35,6 +35,44 @@ function hs_layouts_plugin_filter_block__layout_builder_alter(&$definitions, &$e
   }
 }
 
+/**
+ * Implements hook_preprocess().
+ */
+function hs_layouts_preprocess(&$variables, $hook) {
+  // List of patterns used for collection children elements.
+  $collection_patterns = [
+    'pattern_horizontal_card',
+    'pattern_vertical_card',
+    'pattern_vertical_link_card',
+    'pattern_spotlight',
+    'pattern_timeline',
+    'pattern_hero_text_overlay',
+    'pattern_gradient-hero',
+  ];
+  if (in_array($hook, $collection_patterns)) {
```

Comment:
so many nested conditions. use escape conditions to make things more easily readable.

Reply:
Fixed.